src/core/time.d

```diff
@@ -4714,34 +4719,29 @@ double _abs(double val) @safe pure nothrow @nogc
   +/
 string numToString(long value) @safe pure nothrow
```

Comment:
`numToString` does deal with signed integers, so simply using `core.string.unsignedToTempString()` wouldn't cut it, but given that there's a function for handling this in druntime, I agree that we should definitely use that instead. When I wrote `numToString`, I had no clue that the other existed, and I obviously didn't have access to `std.conv.to`. Maybe we should just add `core.string.signedToTempString()` to handle the signed case. In any case, I agree that that should be another PR.